lib/gamnit/depth/particles.nit

```diff
@@ -153,7 +156,7 @@ end
 #
 # This program should be subclassed to create custom particle effects.
 # Either `vertex_shader_source` and `vertex_shader_core` can be refined.
-abstract class ParticleProgram
+class ParticleProgram
```

Comment:
Is it the `This program should be subclassed` that bugs you?

I'll review the doc for the programs when I reorganize them as a list like `sounds`, `all_root_textures`. This would allow to load only programs in use.